src/Committee/CommitteeManager.php

```diff
@@ -27,10 +30,14 @@ class CommitteeManager
     private const COMMITTEE_PROPOSALS_COUNT = 3;
 
     private $registry;
+    private $storage;
```

Comment:
Please don't add new dependencies for one method, create a new service instead. Like `PhotoManager`.